_config.yml

```diff
@@ -3,4 +3,4 @@ title: Reciprocal Space Station
 plugins:
   - jekyll-font-awesome-sass
 
-port: 4500
\ No newline at end of file
+port: 4502
```

Comment:
I don't really know what's going on here, so I'm happy to take your word for this change.

Reply:
It's super weird! The port is the number of the "local host" that I use to build the website locally while I'm making changes. But then due to some mysterious functionality that no one on Stack Exchange seems to know how to fix, once a port is used, it's like, gone forever? And can never be used again for this purpose. So I have to keep just meaninglessly changing the port every time I build the site locally. The parameter has no affect on the online version of the site, so it's meaningless, but harmless.

Annoyingly, I can't just put the `_config.yml` in the `.gitignore`, because we need it! I could also just be careful to not commit these particular changes, but I'd kinda rather not have my local copy fall out of date with the remote. I think there's also some hacky-ish ways to `.gitignore` only a specific part of a file, but I'm inclined not to worry about it.

Anyway, good catch! I wasn't really thinking about this when I committed it.